app/controllers/todos_controller.rb

```diff
@@ -23,6 +23,12 @@ def create
     render json: '', status: :unprocessable_entity
   end
 
+  def delete_all
+    Todo.delete_all
```

Comment:
Maybe we can keep a history of the previous todos, but that in another PR.

Reply:
https://trello.com/c/2rQYUQzQ/26-keep-history-of-deleted-todos